src/components/Accounts/ChargeFeeFine/ChargeFeeFine.js

```diff
@@ -156,7 +156,7 @@ class ChargeFeeFine extends React.Component {
       const newAction = {
         typeAction,
         source: `${this.props.okapi.currentUser.lastName}, ${this.props.okapi.currentUser.firstName}`,
-        createdAt,
+        createdAt: `${this.props.okapi.currentUser.curServicePoint.name}`,
```

Comment:
Since `createdAt` is replaced here with a string, can/should we remove it as a parameter to `newAction` then? 

Furthermore, reading [the story](https://issues.folio.org/browse/UIU-1398), I wonder if this should really be a service-point-id rather than its name as a string:
 
> In order to balance fees/fines (aka balance the cash drawer) we need to instead record the actual Service where the transaction took place.

How is this value used later?

Reply:
Hi @zburke, I already uploaded the corrections.